source/boss/clunkhead/EntityClunkhead.java

```diff
@@ -113,11 +113,13 @@ else if (choose == 2) {
 
 	public void onLivingUpdate() {
 		super.onLivingUpdate();
+			if (ConfigurationHelper.config.get("Music", "Boss Music", 1).getInt()) {
 		--musicTick;
 		if (musicTick == 0) {
 			musicTick = 330;
 			playSound("nevermine:MusicClunkhead", 2.8f, 1.0f);
 		}
+			}
```

Comment:
```suggestion
		if (ConfigurationHelper.config.get("Music", "Boss Music", 1).getInt()) {
			--musicTick;
			if (musicTick == 0) {
				musicTick = 330;
				playSound("nevermine:MusicClunkhead", 2.8f, 1.0f);
			}
		}
```
the formatting is a bit wonky, no?

Reply:
true but a lot of the code in this mod is from what ive seen. plus it should still work regardless

Reply:
fair enough